cli/command/service/opts.go

```diff
@@ -784,6 +814,8 @@ func addServiceFlags(flags *pflag.FlagSet, opts *serviceOptions, defaultFlagValu
 
 	flags.StringVar(&opts.stopSignal, flagStopSignal, "", "Signal to stop the container")
 	flags.SetAnnotation(flagStopSignal, "version", []string{"1.28"})
+	flags.Var(&opts.isolation, flagIsolation, "isolation mode for the service")
```

Comment:
Can you use a capital `I` here for the flag description? We do so for all the other flags